scene/gui/text_edit.cpp

```diff
@@ -1675,6 +1675,10 @@ void TextEdit::_notification(int p_what) {
 				drag_caret_force_displayed = false;
 				queue_redraw();
 			}
+			if (hovered_gutter != Vector2i(-1, -1)) {
+				hovered_gutter = Vector2i(-1, -1);
+				queue_redraw();
+			}
```

Comment:
The hover update is why I added a `queue_redraw()` in CodeEdit's NOTIFICATION_MOUSE_EXIT, so that one shouldn't be needed anymore.